main.go

```diff
@@ -30,14 +31,22 @@ func EndGame() {
 }
 
 func main() {
+	var isFullscreen = flag.Bool("fullscreen", false, "Play fullscreen!")
+	flag.Parse()
+
 	rand.Seed(time.Now().UnixNano())
 	game = tl.NewGame()
 
 	mainLevel := tl.NewBaseLevel(tl.Cell{
 		Bg: tl.ColorBlack,
 	})
 
-	border = NewBorder(80, 30)
+	var width, height int
+	width, height = 80, 30
```

Comment:
Remove line above, change to `width, height := ...`.

Reply:
Fixed!